docs/Configuration-Settings.md

```diff
@@ -123,6 +123,11 @@ The common configuration contains constants which specify paths and the log appe
   <td>How many times to try to reconnect with master.</td>
 </tr>
 <tr>
+  <td>tachyon.metrics.conf</td>
+  <td>null</td>
```

Comment:
This is the file path of the metrics configuration file. By default it is metrics.properties in the conf directory. I'll write the default path here to make it clear. There is a metrics.properties.template in the conf directory similar to other systems. A user need rename it to metrics.properties to enable metrics.